index.html

```diff
@@ -69,288 +68,273 @@ <h1 class="headline__title color-dark">
+							</a>
+							<a href="https://twitter.com/" rel="noopener noreferrer" target="_blank">
+								<img src="images/icons/twitter.svg" alt="Twitter icon" />
+							</a>
+							<a href="http://www.youtube.com/" rel="noopener noreferrer" target="_blank">
+								<img src="images/icons/m.svg" alt="M icon" />
+							</a>
+						</div>
+						<div class="about__actions">
+							<a href="https://github.com/Rysth" class="button button__primary">Get my resume</a>
+						</div>
+					</header>
+					<div class="about__categories">
+						<div class="about__category">
+							<header class="about__header flex-row justify-between align-center">
+								<h3 class="about__subtitle">Languages</h3>
+								<img src="images/icons/arrow-down.svg" alt="Arrow down icon" />
+							</header>
+							<ul class="about__list grid">
+								<li class="about__item">
+									<img src="images/categories/languages/javascript.png" alt="javascript png image" />
+									JavaScript
+								</li>
+								<li class="about__item">
+									<img src="images/categories/languages/html.png" alt="javascript png image" />
+									HTML
+								</li>
+								<li class="about__item">
+									<img src="images/categories/languages/css.png" alt="javascript png image" />
+									CSS
+								</li>
+							</ul>
+						</div>
+						<div class="about__category">
+							<header class="about__header flex-row justify-between align-center">
+								<h3 class="about__subtitle">Frameworks</h3>
+								<img src="images/icons/arrow-right.svg" alt="Arrow right icon" />
+							</header>
+						</div>
+						<hr />
+						<div class="about__category">
+							<header class="about__header flex-row justify-between align-center">
+								<h3 class="about__subtitle">Skills</h3>
+								<img src="images/icons/arrow-right.svg" alt="Arrow right icon" />
+							</header>
+						</div>
+						<hr />
 					</div>
-					<div class="about__category">
-						<header
-							class="about__header flex-row justify-between align-center"
-						>
-							<h3 class="about__subtitle">Languages</h3>
-							<img src="images/icons/arrow-down.svg" alt="Arrow down icon" />
-						</header>
-						<ul class="about__list grid">
-							<li class="about__item">
-								<img
-									src="images/categories/languages/javascript.png"
-									alt="javascript png image"
+				</div>
+			</section>
+
+			<!-- CONTACTS SECTION -->
+
+			<section class="section section__blue section__blue--top" id="contact">
+				<div class="contact padding-4 text-center">
+					<header class="contact__header flex-column">
+						<h2 class="h2 color-white">Contact me</h2>
+						<p class="contact__description color-white">
+							If you have an application you are interested in developing, a feature that you need built or
+							a project that needs coding. I’d love to help with it!
+						</p>
+					</header>
+					<form action="https://formspree.io/f/mwkjdngo" class="contact__form" method="POST">
```

Comment:
- [ Required ] I think it would be nice if the contact form is responsive across small devices e.g.`375px` Currently, the page has a horizontal scroll which is not consistent with the Figma design. Please see the video below and fix this. 

## Video
https://github.com/Rysth/Portfolio-Project/assets/16392046/901506a1-1ff0-4ea9-9f03-9a34661622bd